server/parsing_lib/log_parser.py

```diff
@@ -8,61 +8,128 @@
 
 
 class LogParser(object):
-    """ Handles parsing of all logs from the Mobile API. """
+    """ Handles parsing of all logs from the Mobile API.
+
+    Attributes:
+        parser_info: info required by the parser that is specific to each OS
+    """
+    parser_info = {
```

Comment:
This is more of a style thing, but it may be easier to read if you put this as an androidParsing & iOSParsing object. That way they can be in their own files and can have additions added. Could even do a baseParsing, and have them inherit